HangPerson/HangPerson/main.m

```diff
@@ -11,8 +11,87 @@
 int main(int argc, const char * argv[]) {
     @autoreleasepool {
         
-        // code goes here...
-
+        printf("Let's play Hangman \n \n");
+        
+        char myWord[] = "anatomy";
+        //        char userWord[ ] = "_______\n";
+        char userLetter;
+        
+        int numChances = 5;
+        int guessCorrect = 0;
+        
+        int lengthOfWord = 7 ;
+        int userGuess[7] = {0,0,0,0,0,0,0};
```

Comment:
For this line an the line above you should use sizeof(myWord) or strlen(myWord) instead of hard coding a number. The more general the better.